### PR TITLE
docs: Fix typos and formatting

### LINKED ISSUE #562: Fix docs
Fix typos and formatting issues.

### PR DESCRIPTION
**Pull request by jodh-intel**
Fix spelling mistakes and formatting issues.

Fixes #562.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: documentation/features/cpuset.md [hunk 1]
```diff
@@ -14,27 +14,31 @@ The cpuset CPU cgroup will be applied on two events:
...
+$ docker run -ti --cpus 2 --cpuset 0,1 busybox
+```
  
 This should result with the container limited to the time of 2 CPUs, but is
 only allowed to be scheduled on CPUs 0 and 1.
 
 The following is an example of a similar case with a valid traditional container:
 
-docker run -ti --cpus 2 --cpuset 2,3,4 busybox
+```
```
Comment (grahamwhaley) [inline documentation/features/cpuset.md]: no `sh` or `bash` type on the formatting? Maybe on purpose to skip auto processing (in which case I think I normally make them `sh`, as only `bash` gets auto processed...)

Comment (jodh-intel) [reply documentation/features/cpuset.md]: Yep - updated for `sh`.

**Comment by jodh-intel** [comment]
2 CI's are failing but this is a doc-only PR, so merging...

**Pull request merged**